src/app/components/side-panel.js

```diff
@@ -13,7 +13,6 @@ const css = csjs`
   }
   .swapitTitle {
     margin: 0;
-    text-transform: uppercase;
```

Comment:
Lets discuss with Natalia about this.

Reply:
Natalia doesn't mind if there are uppercase letters so I made uppercased it again